packages/block-editor/src/components/inserter/hooks/use-insertion-point.js

```diff
@@ -39,6 +39,7 @@ function useInsertionPoint( {
 	isAppender,
 	onSelect,
 	shouldFocusBlock = true,
+	selectBlockOnInsert = true,
```

Comment:
Here I'm aiming to ensure backwards compat by setting the default to the same as the hardcoded original value.

Reply:
Should we make a comment for the future?